Remove timeout in async_test for mediacapture-streams tests

qiuzhong opened the pull request:
Remove all the timeout in `async_test` for mediacapture-streams tests.
Affected tests: 12
Before: Pass: 10, Failed: 2
After: Pass: 10, Failed: 2
Related: #11120

qiuzhong committed:
Remove timeout in async_test for mediacapture-streams tests

Remove all the timeout in `async_test` for mediacapture-streams tests.
Affected tests: 12
Before: Pass: 10, Failed: 2
After: Pass: 10, Failed: 2
Related: #11120

foolip commented on mediacapture-streams/MediaStream-add-audio-track.https.html:
```diff
@@ -15,7 +15,7 @@ <h1 class="instructions">Description</h1>
 <script src=/resources/testharnessreport.js></script>
 <script>
-var t = async_test("Tests that adding a track to a MediaStream works as expected", {timeout: 20000}); // longer timeout since requires double user interaction
+var t = async_test("Tests that adding a track to a MediaStream works as expected"); // longer timeout since requires double user interaction
```
Can you remove the comment which refers to the deleted code?

qiuzhong replied:
Oops! Most of the update was done by a script and I didn't notice that unless comment. It's removed now.

foolip commented on mediacapture-streams/MediaStreamTrack-end-manual.https.html:
```diff
@@ -17,7 +17,7 @@ <h1 class="instructions">Description</h1>
 <script src=/resources/testharnessreport.js></script>
 <script>
-var t = async_test("Tests that the video MediaStreamTrack objects are properly ended on permission revocation", {timeout: 20000}); // longer timeout since requires user interaction
+var t = async_test("Tests that the video MediaStreamTrack objects are properly ended on permission revocation"); // longer timeout since requires user interaction
```
remove comment

qiuzhong replied:
Removed.

qiuzhong committed:
Remove the useless comments

Since the `timeout` is removed from `async_test`, the comments are
useless and should be removed.
